### PR TITLE
PCA-25 + PCA-18. Created template twig file and included some data from DB to index page.

### PR DESCRIPTION
**Pull request by r0bertt**
Created template twig file and included some data from DB to index page.

### CODE REVIEW

File: app/controllers/IndexController.php [hunk 1]
```diff
@@ -12,7 +13,9 @@ public function __construct()
     }
 
     public function index() {
-        $this->view('index', ["title" => Config::get("config", 'name')]);
+        $topics = (DB::queryObject("SELECT * FROM topics LIMIT 10"));
+        //var_dump($topics);
+        $this->view('index', ["topics" => $topics]);
     }
 
 }
```
Comment (indruteok) [inline app/controllers/IndexController.php]: add new line

File: resources/views/index.php.twig [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            <td>{{ topic.Title }}</td>
+            <td>{{ topic.CreationDate }}</td>
+            <td>{{ topic.LastEditDate }}</td>
+            <td <button class="btn btn-outline-success btn-sm" type="submit" id="view">view</button></td>
+        </tr>
+        {% endfor %}
+        <tr><td></td><td></td><td></td><td></td></tr> <!-- A gap after the last table's record-->
+        </tbody>
+    </table>
+{% endblock %}
```
Comment (indruteok) [inline resources/views/index.php.twig:30]: again new line

File: resources/views/template.php.twig [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    <!-- Table field for the content-->
+    {% block table %}
+    {% endblock %}
+    <!-- Footer -->
+    <div class="footer navbar-fixed-bottom">
+        <button class="btn btn-outline-success my-2 my-sm-0" type="submit" id="create">Create</button>
+    </div>
+</div>
+</body>
+</html>
```
Comment (indruteok) [inline resources/views/template.php.twig]: and again new line

File: resources/views/components/navbar.html.twig [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+                    <a class="dropdown-item" href="#">Examples</a>
+
+                </div>
+        </ul>
+        <form class="form-inline my-2 my-lg-0">
+            <input class="form-control mr-sm-2" type="search" placeholder="Search" aria-label="Search">
+            <button class="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
+        </form>
+    </div>
+</nav>
```
Comment (indruteok) [inline resources/views/components/navbar.html.twig]: new line

**Review by indruteok — APPROVED** [review]

File: app/controllers/IndexController.php [hunk 5]
```diff
@@ -12,7 +13,9 @@ public function __construct()
     }
 
     public function index() {
-        $this->view('index', ["title" => Config::get("config", 'name')]);
+        $topics = (DB::queryObject("SELECT * FROM topics LIMIT 10"));
+        //var_dump($topics);
```
Comment (Icybrew) [inline app/controllers/IndexController.php]: Remove

File: resources/views/template.php.twig [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+    <title>{{ title }}</title>
+    {% block stylesheet %}{% endblock %}
```
Comment (Icybrew) [inline resources/views/template.php.twig]: Delete line

File: resources/views/template.php.twig [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+</head>
+<body>
+<div class="container">
+    {{ include('components/navbar.html.twig') }}
+    <!-- Table field for the content-->
+    {% block table %}
```
Comment (Icybrew) [inline resources/views/template.php.twig]: Rename 'table' -> 'content'

File: resources/views/index.php.twig [hunk 8]
```diff
@@ -0,0 +1,30 @@
+{% extends "template.php.twig" %}
+
+{% block table %}
```
Comment (Icybrew) [inline resources/views/index.php.twig]: Rename 'table' -> 'content'

File: resources/views/template.php.twig [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"></script>
+    <title>{{ title }}</title>
```
Comment (Icybrew) [inline resources/views/template.php.twig]: Title is null

**Review by Icybrew — CHANGES_REQUESTED** [review]
See code comments

**Review by indruteok — APPROVED** [review]

**Review by MaslaLT — APPROVED** [review]

**Review by Icybrew — APPROVED** [review]

**Pull request merged**